packages/client/src/components/shop/ShopCard.tsx

```
@@ -0,0 +1,313 @@
+        overflow: 'hidden',
+        border: `1px solid ${equipped ? r.border : 'rgba(212,175,55,0.18)'}`,
+        boxShadow: equipped
+          ? `0 0 22px ${r.glow}, 0 6px 20px rgba(0,0,0,0.6)`
+          : `0 6px 20px rgba(0,0,0,0.5)`,
+        cursor: 'pointer',
+        position: 'relative',
+        userSelect: 'none',
+        animation: `shopCardIn 0.4s cubic-bezier(0.34,1.56,0.64,1) both`,
+        animationDelay: `${index * 55}ms`,
+      }}
+    >
```

Biome (2.5.0)

[error] Unexpected event handler on static element.

(lint/a11y/noStaticElementInteractions)